### ISSUE TITLE
cd using fzf without trigger

### ISSUE THREAD

**Issue by himat** [junegunn/fzf#1606]
- Category
    - [ ] fzf binary
    - [ ] fzf-tmux script
    - [x ] Key bindings
    - [ x] Completion
    - [ ] Vim
    - [ ] Neovim
    - [ ] Etc.
- OS
    - [ ] Linux
    - [ x] Mac OS X
    - [ ] Windows
    - [ ] Windows Subsystem for Linux
    - [ ] Etc.
- Shell
    - [ x] bash
    - [ ] zsh
    - [ ] fish

Is it possible to use fzf automatically after typing `cd` in bash?
I could use ctrl+T from the start without typing `cd`, but a lot of times, I've already typed `cd` and I would rather just start typing the file name right after instead of first typing ** to activate the trigger.

**Comment by shaleh** [comment]
add a `fcd` alias which does what you want. `fcd<enter>` would launch the chooser and go whereever you picked.

**Comment by himat** [comment]
Actually I found the behavior I want.
`$ fjksdl<alt-c>` works because no matter what I have typed at the terminal already, it just CDs into whatever directory I pick in the fzf chooser, and ignores any characters typed before the `alt-c`
But, the issue I have is when I want to use the `ctrl-t` binding.
`$ jfdkl<ctrl-t>` - this only pastes what I choose from the chooser after the text I already have typed.

Is it somehow possible for the `ctrl-t` command to also do the same thing - just replace all of what is currently at the terminal prompt with the chosen option?
